src/main/webapp/app/communication/posting-content/enlarge-slide-image/enlarge-slide-image.component.html

```
@@ -1,3 +1,4 @@
-<div>
+<div class="enlarge-image-container">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)